test/layers.jl

```diff
@@ -214,6 +210,6 @@ video = Video(600, 600)
 
     act!(l5, Action(opacity_anim, setopacity()))
 
+    Javis.preprocess_frames!([vid.objects..., Javis.flatten(vid.layers)...])
     @test_throws ErrorException Javis.get_layer_frame(vid, l5, 6)
```

Comment:
So this method should throw an error due to `act!(l5, Action(opacity_anim, setopacity()))`
   `Frame range for the layer's action might be missing`